include/deal.II/fe/fe_nedelec_sz.h

```diff
@@ -0,0 +1,221 @@
+            dealii::internal::FEValues::FiniteElementRelatedData<dim, dim> &/*output_data*/) const;
+
+  virtual void
+  fill_fe_values (const typename Triangulation<dim,dim>::cell_iterator           &cell,
+                  const CellSimilarity::Similarity                                     cell_similarity,
+                  const Quadrature<dim>                                               &quadrature,
+                  const Mapping<dim,dim>                                         &mapping,
+                  const typename Mapping<dim,dim>::InternalDataBase              &mapping_internal,
+                  const dealii::internal::FEValues::MappingRelatedData<dim, dim> &mapping_data,
+                  const typename FiniteElement<dim,dim>::InternalDataBase        &fedata,
+                  dealii::internal::FEValues::FiniteElementRelatedData<dim, dim> &data) const;
+
+
+  virtual void
+  fill_fe_face_values (const typename Triangulation<dim,dim>::cell_iterator           &cell,
+                       const unsigned int                                              face_no,
+                       const Quadrature<dim-1>                                        &quadrature,
+                       const Mapping<dim,dim>                                         &mapping,
+                       const typename Mapping<dim,dim>::InternalDataBase              &mapping_internal,
+                       const dealii::internal::FEValues::MappingRelatedData<dim, dim> &mapping_data,
+                       const typename FiniteElement<dim,dim>::InternalDataBase        &fedata,
+                       dealii::internal::FEValues::FiniteElementRelatedData<dim, dim> &data) const;
+
+  virtual void
+  fill_fe_subface_values (const typename Triangulation<dim,dim>::cell_iterator           &cell,
+                          const unsigned int                                              face_no,
+                          const unsigned int                                              sub_no,
+                          const Quadrature<dim-1>                                        &quadrature,
+                          const Mapping<dim,dim>                                         &mapping,
+                          const typename Mapping<dim,dim>::InternalDataBase              &mapping_internal,
+                          const dealii::internal::FEValues::MappingRelatedData<dim, dim> &mapping_data,
+                          const typename FiniteElement<dim,dim>::InternalDataBase        &fedata,
+                          dealii::internal::FEValues::FiniteElementRelatedData<dim, dim> &data) const;
+                          
+                          
+  // DERVIED INTERNAL DATA - CAN BE USED TO STORE PRECOMPUTED INFO
+  // E.G. Once dim is known we can compute sigma, lambda, and combinations.
+  class InternalData : public FiniteElement<dim,dim>::InternalDataBase
```

Comment:
Comments like this (and similarly below) should be `/** ... */` doxygen style so that they show up in the generated documentation